File: vlc-android/src/org/videolan/vlc/android/AboutActivity.java
package org.videolan.vlc.android;

import android.app.Activity;
import android.os.Bundle;

public class AboutActivity extends Activity {
	
	public final static String TAG = "VLC/AboutActivity";

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		setContentView(R.layout.about);
		super.onCreate(savedInstanceState);
	}

}

File: application/television/src/main/java/org/videolan/television/ui/AboutActivity.kt
package org.videolan.television.ui

import android.os.Bundle
import androidx.fragment.app.FragmentActivity
import org.videolan.resources.util.applyOverscanMargin
import org.videolan.television.R
import org.videolan.vlc.gui.helpers.UiTools

class AboutActivity : FragmentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.about)
        UiTools.fillAboutView(this, window.decorView.rootView)
        applyOverscanMargin(this)
    }
}
